lib/tizen_emulator.dart

```diff
@@ -62,30 +70,39 @@ class TizenEmulatorManager extends EmulatorManager {
     }
     if (!_tizenEmulators.canLaunchAnything) {
       return CreateEmulatorResult(name,
-          success: false,
-          error: 'emulator manager is missing from the Tizen Studio');
+          success: false, error: 'Unable to locate Tizen Emulator Manager.');
     }
 
-    final PlatformImage platformImage = await _getPreferredPlatformImage();
+    final PlatformImage platformImage = _getPreferredPlatformImage();
     if (platformImage == null) {
       return CreateEmulatorResult(name,
           success: false,
           error:
               'No suitable Tizen platform images are available. You may need to install these'
-              ' using the package manager.');
+              ' using the Tizen Package Manager.'
+              'Visit the link below to see how to create Tizen platform images for emulators:'
+              ' https://docs.tizen.org/application/tizen-studio/setup/update-sdk/#installing-additional-packages');
     }
-    final RunResult runResult = await _processUtils.run(<String>[
-      getEmCliPath(),
-      'create',
-      '-n',
+
+    // throwOnError: when should we use or not use?
```

Comment:
If `throwOnError` is true and the process terminates with non-zero exit code, a process exception will be thrown which can be caught by a try-catch block (otherwise it will abort the tool's execution). Whether you should use `throwOnError` or not depends on how you want to control the execution flow. If you don't want the exception to be triggered, you can manually check if the process exited successfully using the value `result.exitCode`. There are some examples in `tizen_device.dart` and other places.

In this specific case, `throwOnError` should be false (default).

Reply:
Should have erased the comment there, but I did want to know about the considerations in making the decision, thanks!